tasks/install.yml

```diff
@@ -12,15 +12,16 @@
     shell: "/sbin/nologin"
     group: prometheus
     createhome: false
-    home: /tmp
+    home: "{{ prometheus_db_dir }}"
 
 - name: create prometheus data directory
   file:
     path: "{{ prometheus_db_dir }}"
     state: directory
+    mode: 0755
     owner: prometheus
     group: prometheus
-    mode: 0755
+    recurse: true
```

Comment:
Won't this cause all files inside `{{ prometheus_db_dir }}` to have 0755 permissions?

Reply:
No, it doesn't recurse down into the tree, but creates the parent directories (if they don't exist). Think of `mkdir -p`.

Reply:
Hm, I might have been misinformed. I'm confirming against the source code... one sec.

Reply:
https://docs.ansible.com/ansible/latest/modules/file_module.html Says:

> recursively set the specified file attributes (applies only to directories)

Reply:
Yeah, you're right. `state: directory` already creates intermediate directories.

Reply:
If I change this back, there's no difference to #170, is it? In this case, please abandon this PR and merge #170 :-)

Reply:
Merged #170. Thanks for all the testing! :+1: